### DISCUSSION TITLE
Latex packages

### DISCUSSION THREAD

**Discussion by Jafanovic** [artisticat1/obsidian-latex-suite#135] [Ideas]
Hello, 

I like math macros defined in diffcoeff package. Is there any posibility in the future to somehow define own set of macros or import some latex packages? For example diffcoeff.

**Comment by superle3** [comment]
No, for extended functionality of mathjax see the extended mathjax plugin. Its technically not latex but mathjax which is a subset of latex only targeting math.

This plugin only tries to enhance the writing part and nothing else (to keep scope small)
